walk-in-interview/src/main/java/com/google/job/data/Job.java

```diff
@@ -0,0 +1,18 @@
+package com.google.job.data;
+
+public final class Job {
+    // TODO(issue/9);
+    private String jobName;
```

Comment:
Mark this as private final String as we're not modifying it. Unless the public Job() constructor below is required?